serverless/installing_serverless/removing-openshift-serverless.adoc

```diff
@@ -0,0 +1,20 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="removing-openshift-serverless"]
+= Removing {ServerlessProductName}
+include::modules/common-attributes.adoc[]
+:context: removing-openshift-serverless
+
+toc::[]
+
+This guide provides details of how to remove the {ServerlessOperatorName}, as well as other {ServerlessProductName} components.
+
+== Removing the {ServerlessOperatorName}
+
+Before you can remove other {ServerlessProductName} components, such as Knative Serving or Eventing, you must remove the {ServerlessOperatorName}.
```

Comment:
I think it should be the other way around? (You should delete KnativeServing first, then the serverless-operator.)